documentation/jira-guidelines.md

```diff
@@ -0,0 +1,132 @@
+- Completed
+
+### Subtasks
+
+Een Figma component story heeft in principe de volgende Subtasks:
+
+- Research
+  - Check status huidige component: Fabrique / onze Figma, onze Design System website.
+  - Kijk wat er beschikbaar is vanuit NL Design System.
+  - Zoek vergelijkbare ontwerpen in bijv. The Component Gallery
+  - Zet op een rij wat het component allemaal kan en hoe het eruit ziet.
+  - Ga na wat er aan a11y nodig is, bijv. Amsterdam TamTam, Fronteers Slack
+  - Betrek het perspectief van een front-ender en wellicht andere rollen (huisstijl, redactie?).
+  - Bepaal de naamgeving.
+- Design (eventueel meerdere Subtaken)
+  - Werk alle varianten van het component uit in Figma en zet ze overzichtelijk op een rij.
+  - Let op hover, focus states.
+  - Worden transities geanimeerd?
+  - Besteed ook aandacht aan de niet-visuele UI: hoe wordt het voorgelezen, hoe werkt toetsenbordbediening?
+  - Voeg een aantal voorbeelden toe met realistische content. Deze worden overgenomen in Storybook.
+  - Houd rekening met veel en weinig content, responsive gedrag, pointer versus touch, inzoomen van tekst etc.
+  - Vink de Definition of Done af die binnen Figma bestaat.
+- Review
+  - Geef een presentatie aan het team (30 minuten, ad hoc inplannen, kan remote)
+  - Leg de ontwerpkeuzes en overwegingen uit, beantwoord vragen, verzamel feedback
+  - Rond het design af (die Subtask blijft tijdens de Review op Doing staan) naar aanleiding van review
+- Describe
+  - Schrijf documentatie in Markdown / Platte tekst, op basis van de info in de PBI.
+  - Structuur, \* = verplicht:
+    - Rationale \*
+    - UI Description \*
+    - When (not) to use \*
+    - What to consider
+    - Alternatives
+    - Examples \*
+    - WCAG guidelines
+    - Glossary
+    - References
+  - Dit komt letterlijk in Storybook.
+  - Waar tot die tijd bewaren? Figma? OneNote?
+- Publish
+  - Schrijf de change log.
+  - Publiceer het component naar de shared library.
+  - Versienummer?
+  - Waar communiceren?
+- Prepare development
+  - Zet een PBI klaar voor het React component.
+  - Plan een losse refinement in of zet het op de agenda van de aankomende.
+
+Het sprintbord heeft drie kolommen: To do, Doing, Done. Elke Subtask begint in To do. Publish kan pas in To do als de vier voorgaande Done zijn.
+
+## PBI voor bijbehorend React component
+
+### Description
+
+In principe kunnen twee linkjes hier genoeg zijn?
+
+- Link naar Figma
+- Link naar de Figma PBI
+
+Al zullen bijv. Acceptatiecriteria en Oplossingsrichting ook wel specifieke inhoud voor front-end kunnen hebben. Zo ja, toevoegen.
+
+### Workflow
+
+Zo veel mogelijk zoals bij Figma component, mutatis mutandis.
+
+### Subtasks
+
+- Research
+  - Gebruiken we het NLDS community component?
+  - Vind inspiratie online, bijv. The Component Gallery.
+- Development (eventueel meerdere Subtaken)
+  - Design tokens
+  - Class names
+  - React component
+  - Accessibility
```

Comment:
Might be nitpicking, but I'd like to put a11y first in this list. A11y and semantics should define the structure of the component, then you should setup the component in React. Only after the component is functional, you should start styling it. So maybe something like:

- Accessibility
- React component
- Class names
- Design tokens
- Unit tests
- Documentation

Reply:
Well by now you know I love nitpicking :)

I think setting these items op will be iterative anyway, but I can’t be against putting accessibility on top. Applied your suggestion.